spec/features/accessing_invalid_files_spec.rb

```diff
@@ -12,10 +12,11 @@
   describe "for a page I don't have access to" do
     let(:user) { create(:user) }
     let(:facility) { create(:facility) }
+    let(:reservation) { create(:purchased_reservation) }
 
     it "renders a 403 as html" do
       login_as user
-      visit "#{I18n.t('facilities_downcase')}/list.pdf"
+      visit "orders/#{reservation.order.id}/order_details/#{reservation.order_detail.id}/reservations/#{reservation.id}.ics"
```

Comment:
I'm wondering, since we're not sure why the .ics endpoints differ from the .pdf etc ones, is it worth testing both cases?

Reply:
yeah, probably a good idea to leave the previous spec in place and add this one.